manifests/2.15.0/opensearch-2.15.0.yml

```diff
@@ -5,7 +5,7 @@ build:
   version: 2.15.0
 ci:
   image:
-    name: opensearchstaging/ci-runner:ci-runner-centos7-opensearch-build-v3
+    name: opensearchstaging/ci-runner:ci-runner-al2-opensearch-build-v1
```

Comment:
Thanks @peterzhuamazon, I assume this is the dockerfile right https://github.com/opensearch-project/opensearch-build/blob/main/docker/ci/dockerfiles/current/build.al2.opensearch.x64.arm64.dockerfile ?

Reply:
Yes